h2o-core/src/main/java/water/H2O.java

```diff
@@ -2139,6 +2137,16 @@ public static void main( String[] args ) {
     Log.debug("    Start GA: " + (time12 - time11) + "ms");
   }
 
+  /** Find PID of the current process, use -1 if we can find the value */
+  private static void initializePID(){
```

Comment:
Suggestion: remove the side effect of this method; instead changing the static field PID directly return the pid as a result of the method. Set the PID either in main method or in a static initialization block.